app/views/layouts/_analytics.html.erb

```diff
@@ -0,0 +1,17 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', '<%= analytics_tracking_id %>', 'auto');
+
+  ga('set', 'anonymizeIp', true);
+  ga('send', 'pageview');
+
+  <% if content_for?(:transaction_data) %>
```

Comment:
Please remove this IF. It is not in the C100 anymore (I think you may have an old checked out copy locally).
We don't use this GA functionality so no need to have this code.